FreeRTOS/cpp11_gcc/thread.cpp

```diff
@@ -28,6 +29,9 @@
 namespace free_rtos_std
 {
   extern Key *s_key;
+
+  size_t stacksize_lock_section::_stackWordCount{
```

Comment:
Thats a good point! On a second thought, it might be the best to use the FreeRTOS `configSTACK_DEPTH_TYPE` type for the stack size. I have now pushed this change.